### DISCUSSION TITLE
BIcep version 0.14.46?

### DISCUSSION THREAD

**Discussion by mariusostby** [actions/virtual-environments#7099] [Installed software]
The latest version is [0.14.46](https://github.com/Azure/bicep/releases/tag/v0.14.46)

Version included in images for ubuntu-latest and 20.04 as per update [20230206](https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20230206.1) is [0.14.6](https://github.com/Azure/bicep/releases/tag/v0.14.6)

Bicep v 0.14.46 is actually newer than 0.14.6 - and 0.14.6 has a few issues.

Any chance next image update could include the latest bicep version?

**Comment by erik-bershel** [comment]
Hey @mariusostby! Latest available Bicep will be included to a next image. At that moment it is [0.14.46](https://github.com/Azure/bicep/releases/tag/v0.14.46) as you said. But in case Bicep developers release new version of tool as a latest we will include that new one version.
